data-es6.js

```diff
@@ -508,19 +508,20 @@ exports.browsers = {
   safari71_8: {
     full: 'Safari',
     short: 'SF 7.1,<br>SF 8',
-    obsolete: false
+    obsolete: true
   },
   safari9: {
     full: 'Safari',
     short: 'SF 9',
+    obsolete: false,
```

Comment:
We mark 2 latest versions of Safari as stable. This line was added just to make it easier to change it to `true` later